app/views/announcements/index.html.erb

```diff
@@ -1,19 +1,16 @@
-<div id="announcements" class=" p-4">
-
-  <% if notice %>
```

Comment:
Was this notice alert deleted intentionally? If i create an announcement will it flash the success message in a bootstrap alert like it does in admin tools and dashboard? I know this is also the wrong implementation, we should handle if the notice is not success and show it as alert-danger.

Reply:
yes, was done intentionally. I moved the flash message to the authenticated layout so that there would be no need to have it in each individual page.

Reply:
I did not notice that, that is a nice change. Helps keep our code more abstract/reused. Looks good